less/mixins.less

```diff
@@ -1,30 +1,24 @@
-// Mixins Here
+.btn-style {
```

Comment:
The only thing I would do differently here is I would put parens after your mixin name -- I don't think it's necessary, but it would tell the next developer who looks at your code that for sure this is a mixin
```
.img-style() {
 //mixin code here
}

```